### PR TITLE
blog: "What's new in Svelte" April newsletter

### PR DESCRIPTION
**Pull request by DreaminDani**
Let me know if I missed anything!

### CODE REVIEW

**Comment by kyoshino** [comment]
Can you please add my project, [Sveltia CMS](https://github.com/sveltia/sveltia-cms), under **Apps & Sites built with Svelte**? Thanks!

File: sites/svelte.dev/content/blog/2023-04-01-whats-new-in-svelte-april-2023.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+- Support `<!-- svelte-ignore ... -->` on components ([#8082](https://github.com/sveltejs/svelte/issues/8082))
+- Inputs in a `bind:group` will clear when their value is set to `undefined` (**3.56.0**, [#8214](https://github.com/sveltejs/svelte/issues/8214))
+- `<input>` values will now persist when swapping elements with spread attributes in an `{#each}` block (**3.56.0**, [#7578](https://github.com/sveltejs/svelte/issues/7578))
+- Better warnings across the board - from `noreferrer` to `aria` rules (**3.56.0**)
+- Add <svelte:document> (**3.57.0**, [#3310](https://github.com/sveltejs/svelte/issues/3310))
+- The `style:` directive will now take precedence over a `style=` attribute (**3.57.0**, [#7475](https://github.com/sveltejs/svelte/issues/7475))
+- CSS units are now supported in the `fly` and `blur` transitions (**3.57.0**, [#7623](https://github.com/sveltejs/svelte/pull/7623), [Docs](https://svelte.dev/docs#run-time-svelte-transition))
+
+*for all the changes to the Svelte compiler, including unreleased changes, check out the [CHANGELOG](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md).
```
Comment (benmccann) [inline sites/svelte.dev/content/blog/2023-04-01-whats-new-in-svelte-april-2023.md:31]: Is the `*` meant to be a bullet? Maybe it should have a space after it if so? And maybe use `-` for consistency with above?

Comment (DreaminDani) [reply sites/svelte.dev/content/blog/2023-04-01-whats-new-in-svelte-april-2023.md:31]: No. It's supposed to be footnote. I'll just change it to a regular sentence 😅

**Comment by DreaminDani** [comment]
@benmccann / @dummdidumm i think this is ready to merge :)

File: sites/svelte.dev/content/blog/2023-04-01-whats-new-in-svelte-april-2023.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+- You can now get all cookies for a request with `cookies.getAll` (**1.10.0**, [#9287](https://github.com/sveltejs/kit/pull/9287), [Docs](https://kit.svelte.dev/docs/types#public-types-cookies))
+- Easily manage the submission status of (multiple) forms with the new exposed `submitter` parameter in `use:enhance` (**1.12.0**, [#9425](https://github.com/sveltejs/kit/pull/9425), [Docs](https://kit.svelte.dev/docs/types#public-types-submitfunction))
+- The default error page now has dark mode styles (**1.13.0**, [#9460](https://github.com/sveltejs/kit/pull/9460))
+
+---
+
+## Community Showcase
+
+**Apps & Sites built with Svelte**
+- [Peerbeer](https://peer.beer/) let's you share files peer-to-peer (p2p) without any third parties or data limits
```
Comment (dummdidumm) [inline sites/svelte.dev/content/blog/2023-04-01-whats-new-in-svelte-april-2023.md:43]:
```suggestion
- [Peerbeer](https://peer.beer/) lets you share files peer-to-peer (p2p) without any third parties or data limits
```

File: sites/svelte.dev/content/blog/2023-04-01-whats-new-in-svelte-april-2023.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+- The `style:` directive will now take precedence over a `style=` attribute (**3.57.0**, [#7475](https://github.com/sveltejs/svelte/issues/7475))
+- CSS units are now supported in the `fly` and `blur` transitions (**3.57.0**, [#7623](https://github.com/sveltejs/svelte/pull/7623), [Docs](https://svelte.dev/docs#run-time-svelte-transition))
+
+For all the changes to the Svelte compiler, including unreleased changes, check out the [CHANGELOG](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md).
+
+## What's new in SvelteKit
+- You can now get all cookies for a request with `cookies.getAll` (**1.10.0**, [#9287](https://github.com/sveltejs/kit/pull/9287), [Docs](https://kit.svelte.dev/docs/types#public-types-cookies))
+- Easily manage the submission status of (multiple) forms with the new exposed `submitter` parameter in `use:enhance` (**1.12.0**, [#9425](https://github.com/sveltejs/kit/pull/9425), [Docs](https://kit.svelte.dev/docs/types#public-types-submitfunction))
+- The default error page now has dark mode styles (**1.13.0**, [#9460](https://github.com/sveltejs/kit/pull/9460))
+
```
Comment (dummdidumm) [inline sites/svelte.dev/content/blog/2023-04-01-whats-new-in-svelte-april-2023.md:37]:
```suggestion
- You can now omit types on all methods and variables with special meaning to SvelteKit and still benefit from full type safety! Read more about it in the [announcement blog post](https://svelte.dev/blog/zero-config-type-safety)

```

**Review by dummdidumm — APPROVED** [review]
Thank you, as always 😊

**Pull request merged**